### PR TITLE
Misc. minor fixes collected during review

### PR DESCRIPTION
**Pull request by msdemlei**
I'd hope these are all uncontroversial, although some are actually a bit more than just fixing typos (e.g., proper formulae for a few transformations).

### CODE REVIEW

**Comment by mbtaylor** [comment]
@msdemlei, this is branched a long way back (29 commits, 4 months) from the current master, any good reason for that?  It would be easier to review the changes if was based from the current master.

**Comment by msdemlei** [comment]
On Thu, Nov 11, 2021 at 01:44:49AM -0800, Mark Taylor wrote:
> @msdemlei, this is branched a long way back (29 commits, 4 months)
> from the current master, any good reason for that?  It would be
> easier to review the changes if was based from the current master.

Ah, *dang*, I had pulled from the wrong upstream *again*.

The PR should now be against the current master.  And I'll have to
revise my RFC comments.

**Comment by mbtaylor** [comment]
I've got some corrections to your corrections.  I could note them here, but (I think) if you click the "Allow edits from maintainers" button bottom right of this PR I can apply them directly (see [here](https://stackoverflow.com/questions/20928727/adding-commits-to-another-persons-pull-request-on-github)).

**Comment by msdemlei** [comment]
On Thu, Nov 11, 2021 at 03:22:14AM -0800, Mark Taylor wrote:
> I've got some corrections to your corrections.  I could note them
> here, but (I think) if you click the "Allow edits from maintainers"
> button bottom right of this PR I can apply them directly (see

I'm allowing edits by maintainers already, so do try to push.  In
case github won't let you, I'm old-school enough to be able to deal
with diffs per mail.

**Comment by mbtaylor** [comment]
Oops you're right, sorry, I had my remotes set up wrong.  Corrections applied.

File: EPNTAP.tex [hunk 1]
```diff
@@ -2501,8 +2504,8 @@ \subsection{Registering EPN Data Collections and Services}
 exposed through protocols different from TAP.  Normally, however, the
 tableset will be contained in a VODataService \xmlel{CatalogService}
 record with a TAP capability, and this capability will be an auxiliary
-capability as per DDC \citep{2019ivoa.spec.0520D}.  For one-table
-services a full TAPRegExt \citep{2012ivoa.spec.0827D} capability is also
+capability as per DDC \citep{2019ivoa.rept.0520D}.  For one-table
```
Comment (mbtaylor) [inline EPNTAP.tex:2507]: The new ID here, `2019ivoa.rept.0520D`, is no longer defined.  I don't know whether that means the edit was wrong or an ivoatex bib update is required.

**Review by mbtaylor — APPROVED** [review]
Looks OK apart from citation issue separately noted.

**Comment by msdemlei** [comment]
On Thu, Nov 11, 2021 at 05:52:37AM -0800, Mark Taylor wrote:
> -services a full TAPRegExt \citep{2012ivoa.spec.0827D} capability is also
> +capability as per DDC \citep{2019ivoa.rept.0520D}.  For one-table
> 
> The new ID here, `2019ivoa.rept.0520D`, is no longer defined.  I don't know whether that means the edit was wrong or an ivoatex bib update is required.

No, my bad, sorry for missing that.  We at some point changed the
bibstem of endorsed notes from "rept" (for notes) to "spec" (for
something reviewed), and somehow I must have used an outdated ivoatex
for building the doc (oh, the magic of submodules never fails to
confuse me).  It's fixed now.

Thanks for reviewing!

**Comment by Erard** [comment]
Many thanks for the many corrections. I just wonder why you removed the details of spectral_sampling_step conversions. The point here was to provide data providers with a clear conversion scheme and adapted coefficients. This was intended to  avoid conversion issues, and this actually helped.

**Comment by msdemlei** [comment]
On Fri, Nov 12, 2021 at 01:44:56AM -0800, Stéphane Erard wrote:
> Many thanks for the many corrections. I just wonder why you removed
> the details of spectral_sampling_step conversions. The point here
> was to provide data providers with a clear conversion scheme and
> adapted coefficients. This was intended to  avoid conversion
> issues, and this actually helped.

For the sampling steps, I haven't dropped the conversions but turned
them into proper equations (around l. 939).  I've just removed the
separate formulae for wavelengths in nm and in um, as I'd consider it
a bit insulting to our readers to insinuate they may require
assistence when converting between nm and um.

The other conversion parts I'd indeed like to remove (but that
doesn't happen in in this PR because I suspected that might not be
uncontentious; see my RFC comments), as I don't think we should
have simple variations of c = λ ν in a spec.  But if we keep them,
I'd say they should be proper formulae, too.

**Comment by mbtaylor** [comment]
I wasn't feeling personally insulted...  since the original authors are likely closer to the intended readership than the later editors I'd support @Erard if he wants to change the spectral_sampling_step conversions etc back to something closer to their original form.

**Comment by Erard** [comment]
It may be my teaching practice… but I think it never hurts to provide too much information. Plus, some of the people involved are not specialists, and providing a clear reference did help in the past (and it didn't prevent from getting data 6 orders of magnitudes in error!). So yes, I'd keep it.

**Comment by msdemlei** [comment]
On Fri, Nov 12, 2021 at 08:09:17AM -0800, Stéphane Erard wrote:
> It may be my teaching practice… but I think it never hurts to
> provide too much information. Plus, some of the people involved are

Oh, yes, it does, in a spec in any case.  Including too much
non-normative text in a spec will make it clumsy and confusing.

But in this case I'm, really, mainly concerned about the style --
just dumping a few expressions in typewriter text without any
comment, I think, will only help people who don't need that help.
Everyone else will either be baffled, ignore the material or, worst,
just paste the expressions when they shouldn't.

> not specialists, and providing a clear reference did help in the
> past (and it didn't prevent from getting data 6 orders of
> magnitudes in error!). So yes, I'd keep it.

Well, count me as unconvinced, but if you'd like these kinds of
expressions with units in, let's try to find some way we can both
live with.  I've just committed an another line that then has a
conversion with a unit -- I still find that really clumsy, but that I
could live with.  What do you think of it?  In case you'd like to see
it formatted without fiddling with git remotes, there's
http://docs.g-vo.org/EPNTAP.pdf, it's on p. 17/18.

**Comment by BaptisteCecconi** [comment]
I agree that typewriter-style fomulae are not the best option here. I would advocate (again) to keep an operational value of `c` to be used on provider's and client's sides.

**Comment by Erard** [comment]
I'm not sure we're talking about the same thing. I'm not a big fan of typewriter style either (that came from Confluence limited styles), but providing the reference c value and powers of 10 proved to be a big time saver in many cases. Some  people actually working on the services have limited knowledge of spectroscopy - I won't give names :)

**Comment by BaptisteCecconi** [comment]
ok, then we agree 👍

**Comment by msdemlei** [comment]
On Mon, Nov 15, 2021 at 01:55:16AM -0800, Baptiste Cecconi wrote:
> About the spectral unit conversions, we wanted to have an
> operational value for conversions from frequencies to wavelengths.
> The goal is to tell providers and client what value of `c` shall be
> used, so that conversions goes back an forth smoothly, not loosing
> precision after a few rounds of conversions.

Ah... well, frankly, I have a hard time imagining that having c to
less than a percent precision could operationally matter *anywhere*
(as I expect errors in the data and discovery fuzziness dominating by
far), but if that is your concern, I'd say a preface-like thing
saying

  For the purposes of this version of EPN-TAP, we assume

  c = 2.99...
  1 AU = 1.49...
  (whatever else)

-- a bit like section 4 through 6 in RegTAP,
https://ivoa.net/documents/RegTAP/20191011/REC-RegTAP-1.1.html, would
be a lot better than repeating c in three or four sections.  Yes, c
won't change any more, but still.

Is that a vote against my approximate formula with the units?  You'd
really prefer all the digits of c?

**Comment by BaptisteCecconi** [comment]
Yes, that would serve the purpose, indeed. I vote for the preface option for all numerical values used in the standard.

**Comment by Erard** [comment]
Yes - we need exact coefficients to convert back and forth in the portal. Most optical services are wavelength-based and need be converted to frequency in the service table, but are then searched again in wavelength from the portal. An error of 10% results in data loss (like : I'm looking for spectra covering the 3.3 - 3.6 µm range for organic materials) - and some providers are extremely picky when it comes to presenting their capacities. Plus, it doesn't hurt to use the exact values.

**Comment by Erard** [comment]
But I prefer to keep the values close to where they are used - no need to make life more difficult to anybody.

**Comment by msdemlei** [comment]
On Mon, Nov 15, 2021 at 04:35:16AM -0800, Stéphane Erard wrote:
> But I prefer to keep the values close to where they are used - no
> need to make life more difficult to anybody.

The trouble is that it's not always easy to know in advance what's
more difficult -- having a quick reference in one place or having
things where they're needed but strewn across the document and
repeated in potentially many places.

Proposal: Let's merge this PR, and I'll make another one beautifying
the other conversions and provide a quick reference to the constants
involved.  What quantities except c and AU would we need?

**Comment by msdemlei** [comment]
Ok, I've backed out the Conversions reformatting for now so this can get merged.  I'll do a separate PR that will attempt to improve the presentation, clarity, and usability of the conversion examples.

**Pull request merged**